chessington/ui/app.py

```diff
@@ -6,53 +6,56 @@
 from chessington.engine.board import Board
 from chessington.engine.data import Square
 
-app = Flask(__name__)
+def create_app():

```

Comment:
(just makes the set up for Selenium slightly more akin to the M3 exercise which assumes this has been done, shouldn't affect anything else)